src/main/kotlin/org/kamiblue/client/setting/settings/AbstractSetting.kt

```diff
@@ -48,4 +48,6 @@ abstract class AbstractSetting<T : Any> : Nameable {
         val parser = JsonParser()
     }
 
+    fun isModified() = this.value != this.defaultValue
```

Comment:
This can be a getter instead of a method, and this should be placed with `isVisible`